website/src/pages/index.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import Layout from "@theme/Layout";
+import * as icons from "../icons";
+
+import styles from "./index.module.css";
+
+type FeatureItem = {
+  title: string;
+  Icon: React.ComponentType<React.SVGProps<SVGSVGElement>>;
+  description: string;
+};
+
+const FeatureList: FeatureItem[] = [
+  {
+    title: "Pub/Sub logging",
+    Icon: icons.Robot,
+    description:
+      "MCAP is ideal for storing multiple channels of timestamped log data, such as pub/sub messages or multimodal sensor data.",
+  },
+
+  {
+    title: "Serialization agnostic",
+    Icon: icons.DrawerEnvelope,
+    description:
+      "MCAP is a container format, allowing you to record and replay binary messages in any format, such as Protobuf, DDS (CDR), ROS, JSON, etc.",
+  },
+  {
+    title: "High performance writing",
+    Icon: icons.SportsCarConvertible,
+    description:
+      "MCAP utilizes a row-oriented, append-only design. This minimizes disk I/O, and reduces the risk of data corruption during an unclean shutdown. ",
+  },
+  {
+    title: "Self-contained",
+    Icon: icons.ShipmentPackage,
+    description:
+      "MCAP files are fully self-contained, including schemas required to deserialize each channel. Older files always remain readable, even as your codebase evolves.",
+  },
+  {
+    title: "Efficient seeking",
+    Icon: icons.Turntable,
+    description:
+      "MCAP files contain an optional index, allowing for fast, efficient reading, even over a low-bandwidth internet connection.",
```

Comment:
```suggestion
      "MCAP files support optional indexing, allowing for fast, efficient reading, even over a low-bandwidth internet connection.",
```

Reply:
"indexing" sounds like something I would do external to the file (e.g. Data Platform "indexes" mcap files), versus "mcap files contain an index". don't you think?

Reply:
feel free to ignore this if you don't like it